CONTRIBUTING.md

```diff
@@ -5,25 +5,31 @@ completely by volunteers in their spare time, so any contribution--no matter how
 small--is greatly appreciated. In this file, we'll outline how you can
 contribute to the project.
 
+If you have questions after reading this, you can join our Gitter channel via
+the badge at the top of `./README.md`.
+
 ## Overview
 
 Several of the main ways that you can contribute to the project include:
 
-* Updating [the website](https://github.com/xournalpp/xournalpp.github.io)
-  with additional information.
-* Submitting translation improvements via Crowdin (see below).
-* Contributing code changes with pull requests (PRs).
+* [Contributing website improvements](#contributing-website-improvements)
+* [Contributing translation improvements](#contributing-translation-improvements)
+* [Contributing code improvements](#contributing-code-improvements)
+
+## Contributing website improvements
+
+For making changes to the website, please go to [xournalpp/xournalpp.github.io](https://github.com/xournalpp/xournalpp.github.io).
 
-## Contributing translations
+## Contributing translation improvements
 
-If you would like to contribute translations, you can submit improvements to
-[our project on Crowdin](https://crowdin.com/project/xournalpp). The Crowdin
-translations are merged back into the main code periodically, after which they
+We use via [Crowdin](https://crowdin.com) for contributing translation
+improvements. To do this, see [our project on Crowdin](https://crowdin.com/project/xournalpp). The Crowdin
+translations are merged back into the main code periodically, after which, they
```

Comment:
I don't find this change very helpful; the original has a single, unambiguous link to the translation project, whereas the new version requires readers to figure out which one to go to.